integr8ly/integreatly-operator
# INTLY-4327 - Update 3scale operator to use 2.7 3scale

## Description
Updates the 3scale operator to use 3scale 2.7. 

CSV files were copied from https://github.com/3scale/3scale-operator/tree/3scale-2.7.0/deploy/olm-catalog/3scale-operator with edits to point to the `registry.redhat.io` instead of `registry.stage.redhat.io` and removing the old `0.3.0` csv and the `replaces` label

Jira:
 * https://issues.redhat.com/browse/INTLY-4327

## Verification
* Clone this branch
* Run the operator locally against cluster with at least these products installed:
```
operator-sdk up local --namespace=integreatly --operator-flags=--products=rhsso,rhssouser,cloud-resources,3scale
```
* Ensure 3scale gets installed correctly
* Run the `setup-htpass-idp.sh` script and note the outputed password for `customer-admin`
```
./scripts/setup-htpass-idp.sh
```
* Login in to 3scale using the Single Sign On option using the `htpasswd` provider using the `customer-admin` credentials as noted above
* Input account information while keeping username as `customer-admin`
* Link to existing `customer-admin` accont, using the password: `Password1`
* Ensure user signed it correctly and can complete the tutorial for creating an example endpoint after initial login
